packit/cli/propose_downstream.py

```diff
@@ -11,20 +11,38 @@
 import click
 
 from packit.cli.types import LocalProjectParameter
-from packit.cli.utils import cover_packit_exception, iterate_packages, get_packit_api
+from packit.cli.utils import (
+    cover_packit_exception,
+    iterate_packages,
+    get_packit_api,
+)
 from packit.config import pass_config, get_context_settings, PackageConfig
 from packit.config.aliases import get_branches
 from packit.constants import (
     PACKAGE_LONG_OPTION,
     PACKAGE_SHORT_OPTION,
     PACKAGE_OPTION_HELP,
+    CENTOS_STREAM_DISTGIT_URL,
 )
+from packit.config.job_config import JobType
 
 logger = logging.getLogger(__name__)
 
 
 def get_dg_branches(api, dist_git_branch):
     cmdline_dg_branches = dist_git_branch.split(",") if dist_git_branch else []
+
+    # TODO(Gris): Are we support mixing c9s with fedora-all ?
+    if any(
+        job.type == JobType.propose_downstream
+        and any(
+            pkg_conf.dist_git_branches and "c9s" in pkg_conf.dist_git_branches
+            for pkg_conf in job.packages.values()
+        )
+        for job in api.package_config.jobs
+    ):
+        api.config.pkg_tool = "centpkg"
+        api.package_config.dist_git_base_url = CENTOS_STREAM_DISTGIT_URL
```

Comment:
I think we should not touch jobs here (= in packit repo code) at all, as the jobs are service-specific...we should probably rather check the `dist_git_branch` argument, WDYT?

Reply:
In V2 patch, I created two instance of `PackitApi` as workaround. This require better design which I don't have resource to do.